### DISCUSSION TITLE
Are these two games compatible

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#2072] [General]
Are Playdead Inside and Little Nightmares compatible with skyline as of yet? I don't own the edge version. If these two are I might get the edge. The last time the issues section was updated for these two games were 5 months ago.

**Comment by svenem-dos** [comment]
I updated the game Inside.
https://github.com/skyline-emu/skyline-games-list/issues/322

Can't test Little Nightmares for you. (yet)
